SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift

```diff
@@ -65,3 +65,11 @@ extension NSNumberFormatter {
         return decimalFormatter
     }
 }
+
+public extension Int {
```

Comment:
Not really keen on this being on all Int types...

NSTimeInterval + NSDateFormatter provides a good built in solution for this formatting imho